map/framework.hpp

```diff
@@ -700,6 +700,9 @@ class Framework : public PositionProvider,
   bool LoadIsolinesEnabled();
   void SaveIsolinesEnabled(bool enabled);
 
+  bool LoadOutdoorsEnabled();
+  void SaveOutdoorsEnabled(bool enabled);
```

Comment:
Hmm, it means that iOS styles processing works in some other way? ..

Reply:
Styles are properly saved in iOS on master branch.